.github/workflows/frontend.yml

```diff
@@ -77,9 +77,6 @@ jobs:
       - name: Unit Tests
         run: npm run test:unit
 
-      - name: Integration Tests
-        run: npm run test
-
```

Comment:
Instead of disabling integration tests, let's just fix them. Recommend changing port [here](https://github.com/cse403-lemmeknow/lemmeknow/blob/4f329c3c2eb277904d66d867a940bec00e5fa291/frontend/playwright.config.js#L7) to the correct value of `5173`.

That should make the command succeed. Researching how to make it actually test things.

Reply:
Ok here are the steps:
1. rename `frontend/tests/test.js` to `frontend/tests/test.spec.js`
2. add `npx playwright install` as a new step

Reply:
i think i had to create a different test.js file